include/snowflake/client.h

```diff
@@ -18,7 +18,7 @@ extern "C" {
 /**
  * API Name
  */
-#define SF_API_NAME "C API"
+#define SF_API_NAME "ODBC"
```

Comment:
it's temporal - please add todo comment to change before merging to master

Reply:
done